### ISSUE TITLE
custom derivation path

### ISSUE THREAD

**Issue by tezoswakenbake** [airgap-it/beacon-sdk#18]
Hello, i can't connect pair my tezos account to beacon because is in a different derivation path in my ledger X device.

**Comment by pascuin** [comment]
I assume you're referring to not being able to do this in the Beacon Extension?

**Comment by tezoswakenbake** [comment]
> I assume you're referring to not being able to do this in the Beacon Extension?

Yes, my tezos account is in path: 44'/1729'/0'/1 thus it won't pick it up. 

Beacon needs a way for the user to add a custom derivation path.

**Comment by AndreasGassmann** [comment]
This is currently being tested internally and should be out soon.

I'll also move this issue to the extension repo.
